app/components/billing/authorization.js

```diff
@@ -46,7 +46,8 @@ export default Component.extend({
 
   retryAuthorization: task(function* () {
     if (this.requiresSourceAction && this.retryAuthorizationClientSecret) {
-      yield this.stripe.handleStripePayment.perform(this.retryAuthorizationClientSecret);
+      this.stripe.handleStripePayment.perform(this.retryAuthorizationClientSecret);
```

Comment:
It feels to me that `yield` is still required here to wait for the payment task to perform, isn't it?